src/components/playerstats/playerstats.js

```diff
@@ -269,31 +269,10 @@ import ServerConnections from '../ServerConnections';
             });
         }
 
-        if (videoStream.VideoRange) {
+        if (videoStream.VideoRangeType) {
             sessionStats.push({
-                label: globalize.translate('LabelVideoRange'),
-                value: videoStream.VideoRange
-            });
-        }
-
-        if (videoStream.ColorSpace) {
-            sessionStats.push({
-                label: globalize.translate('LabelColorSpace'),
-                value: videoStream.ColorSpace
-            });
-        }
-
-        if (videoStream.ColorTransfer) {
-            sessionStats.push({
-                label: globalize.translate('LabelColorTransfer'),
-                value: videoStream.ColorTransfer
-            });
-        }
-
-        if (videoStream.ColorPrimaries) {
```

Comment:
Removed as unnecessary (for the user)?

Reply:
The stats list is too long and it's better to replace these technical infos with "HDR10/HLG/DOVI".